src/main.rs

```diff
@@ -41,13 +46,13 @@ use crate::settings::*;
 mod clean_text;
 use crate::clean_text::*;
 
-struct State<'a> {
-    voice: Box<SpVoice<'a>>,
```

Comment:
Lifetime parameter no longer needed since the COM initialize method now returns a handle rather than a Rust pointer.